lambda/main.tf

```diff
@@ -39,25 +39,22 @@ variable envvars {
   default = {}
 }
 
-resource "aws_iam_role" "role" {
-  name               = var.name
-  assume_role_policy = <<POLICY
-{
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Action": "sts:AssumeRole",
-      "Principal": {
-        "Service": [
-            "lambda.amazonaws.com"
-        ]
-      },
-      "Effect": "Allow",
-      "Sid": ""
```

Comment:
`Sid` is unnecessary?

Reply:
the raw JSON had it empty, which is a default value for `sid` keyword in `document` block. I'll add `sid = ""` anyway to make it clear but yes, it's not needed.

Reply:
I was just curious, you can leave it as it is.